projects/rocprofiler-systems/source/lib/core/rocprofiler-sdk.cpp

```diff
@@ -436,6 +437,15 @@ get_callback_domains()
         _data.emplace(ROCPROFILER_CALLBACK_TRACING_RCCL_API);
     }
 
+#    if ROCPROFILER_VERSION >= 600
+    // Translate some configuration settings to rocprofiler domains
+    if(config::get_use_ompt() && _version.formatted >= 600)
+    {
+        _data.emplace(ROCPROFILER_CALLBACK_TRACING_OMPT);
```

Comment:
```suggestion
    if(config::get_use_ompt() && _version.formatted >= 600)
    {
        // Translate ROCPROFSYS_USE_OMPT to entry in ROCPROFSYS_ROCM_DOMAINS
        _data.emplace(ROCPROFILER_CALLBACK_TRACING_OMPT);
```
Nit: Just to make comment uniform with the above one for RCCL